elyra/pipeline/registry.py

```diff
@@ -793,8 +793,8 @@ def _get_reader(self, component):
         Find the proper reader based on the given registry component.
         """
         try:
-            component_type = list(component['path'].keys())[0]
-            return self.readers.get(component_type)
+            reader_type = list(component['path'].keys())[0]
```

Comment:
Since the change is not related to the PR title it should be listed in the PR description